lang/perl5/Portfile

```diff
@@ -209,6 +209,17 @@ foreach {perl5.v perl5.subversion perl5.revision perl5.rmd160 perl5.sha256 perl5
         post-build {
             reinplace -E {s/-arch [a-z0-9_]+//g} \
                 ${worksrcpath}/lib/Config_heavy.pl
+            reinplace -E {s/-mmacosx-version-min=[0-9.]+//g} \
+                ${worksrcpath}/lib/Config_heavy.pl
+            reinplace "s|${configure.cc}|/usr/bin/cc|g" \
+                ${worksrcpath}/lib/Config.pm \
+                ${worksrcpath}/lib/Config_heavy.pl \
+                ${worksrcpath}/config.h
+            reinplace -E {s|-isysroot[[:space:]]*[^[:space:]]+\.sdk||g} \
+                ${worksrcpath}/lib/Config_heavy.pl \
+                ${worksrcpath}/config.h
+            reinplace -E {s|-Wl,-syslibroot,[^[:space:]]+\.sdk||g} \
+                ${worksrcpath}/lib/Config_heavy.pl
```

Comment:
Awesome, thank you very much.
I came up with a `sed` expression to do exactly that, but I had some troubles translating it into Tcl ;)

This still leaves one trace of a header file for time in `Config_heavy.pl` with full path hardcoded in, but I have no clue how to get rid of that one.

Reply:
Yes, that's some of the further work I alluded to.